src/Cart.js

```diff
@@ -0,0 +1,55 @@
+// Create components Cart, Item and CartTotal. Cart component displays list of Items and CartTotal.
+// (set items in the state of Cart)
+// sample items = [ {name: 'item1', qty:2, price: 30}, {name: 'item2', qty:5, price: 10}, {name: 'item3', qty:7, price: 120}, ]
+
+
+import React from 'react';
+export default class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      item:[ {name: 'item1', qty:2, price: 30}, {name: 'item2', qty:5, price: 10}, {name: 'item3', qty:7, price: 120}],
+      CartTotal:0
+    };
+  }
+
+  render() {
+
+    return (
+      <div>
+
+        <table>
+          <tbody>
+          <tr>
+            <th>Name</th>
+            <th>Quantity</th>
+            <th>Price</th>
+          </tr>
+
+          {this.state.item.map((item, i) => {
+            this.state.CartTotal += (item.qty*item.price);
```

Comment:
does this work?
state should only be set in constructor
`this.state.CartTotal += (item.qty*item.price);` is equal to `this.state.CartTotal = this.state.CartTotal + (item.qty*item.price);` and changing state directly in render should be avoided